programs/autocrat/src/state/dao.rs

```diff
@@ -16,6 +16,9 @@ pub struct Dao {
     // fail price in order for the proposal to pass
     pub pass_threshold_bps: u64,
 
+    // Proph3t: I don't think we need the burn logic anymore because of AMM
```

Comment:
Makes sense. Ok, I'll remove it.

Reply:
I think we actually still run into the same issue here as with the proposal_instructions, where people can spam them. I'll set it up so that there is a 1 sol fee in part_one that gets repaid in part 2.